apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -79,6 +71,7 @@ export default class QuickView extends React.Component {
         applications: data
       });
     });
+    this.setState({isWorkshopAdmin: getScriptData("props")["isWorkshopAdmin"]});
```

Comment:
Why are we reading script data within the React control? Ideally this should be passed in as a prop, for better composition and testing. Unless we have a good reason otherwise, the entry point js should read all the relevant script data and pass it in as props.

Reply:
OH that would solve my problems with the testing! thank you!